src/BackpackServiceProvider.php

```diff
@@ -260,7 +260,7 @@ public function loadConfigs()
         [
             'backpack' => [
                 'provider'  => 'backpack',
-                'table'     => 'password_resets',
+                'table'     => config('backpack.base.password_resets_table') ?? config('auth.passwords.users.table'),
```

Comment:
Hey @promatik I don't agree keeping this connection with `password.users` in `passwords.backpack`. Let me try to explain what I am thinking here.

I assume this will always default to the Laravel configuration unless specified by developer in config. I don't think we should do that, **I know we are doing** it 🔴 , but still 😞 

What if there is no `passwords.users` ? Developer could have `passwords.admin` and `passwords.guest` instead. 

The purpose of registering a new "user type", in our case `backpack`, is giving the developer the ability to have two separate configs (including table, yes ... I know 😢 ) for different user types of the application, as stated by Laravel:
```php
/*
    |--------------------------------------------------------------------------
    | Resetting Passwords
    |--------------------------------------------------------------------------
    |
    | You may specify multiple password reset configurations if you have more
    | than one user table or model in the application and you want to have
    | separate password reset settings based on the specific user types.
    |
    |
     ... more docs
    */
```
We kinda do that, except we force the developer to use the same password reset table of `users` for `backpack` users. 

I know .. it's "just a table to store some temporary tokens", but what if in `users` password reset table developer add one more non-nullable field, lets say `user_token_salt`, that exists only on the `users` domain and not in `backpack`? 

I agree that we create a config for it, not in a file, in a service provider, that could default **to the first defined entry in `auth.passwords`**, not `users` specifically because that key could be inexistent.

But only that wouldn't solve the whole problem, we still need to provide a away for people that want to have two separate tables.

We can probably do it without a BC by documenting it:
> If you want to have two separate user types with two different password reset tables in your application run `php artisan backpack:password-resets` to publish the migration (`backpack_password_resets` table), and then change in config `backpack.base.password_resets_table` the table name to `backpack_password_resets`.

This is the config we set by default in the service provider if not set by developer in a config file.

We don't publish the migration by default, we keep assuming that developer is fine by using the "defaults".

#### Sum up: 
I think we should have done this from the start, we basically "hijacked" the default `users` password reset table into our `backpack` password reset table. If the developer intended to use both `users` and `backpack` as separate entities he would need to manually create another table for their `users`, because our `backpack` was not configurable. 

The quick win is providing the default config in a service provider, so that we can target the first defined `auth.passwords` entity.

Then provide the sub and instructions on how to create the separate table if needed. 

What you think @promatik ?

@tabacitu anything you think I am missing here? You were the one that implemented the current solution. 

Cheers